prisma/schema.prisma

```diff
@@ -70,3 +72,14 @@ model DeliveryLogLine {
   logId   Int
   log     DeliveryLog @relation(fields: [logId], references: [id])
 }
+
+model Like {
+  likedBy     User     @relation(fields: [likedById], references: [id], onDelete: Cascade)
+  likedById   Int
+  likedPost   Post     @relation(fields: [likedPostId], references: [id], onDelete: Cascade)
+  likedPostId Int
```

Comment:
I'm not a fan of these field names, they should represent the model they connect to.

```js
user       User
userId     Int
post       Post
postId     Int
```

With the above, a prisma query might look like

```js
await prisma.like.findMany({
    where: {
        userId: user.id
    }
})
```

Compare how easy to read that is with this:

```js
await prisma.like.findMany({
    where: {
        likedById: user.id
    }
})
```